integration-tests/scaffolding/extrinsicHelpers.ts

```diff
@@ -1,11 +1,11 @@
 import { ApiPromise, ApiRx } from "@polkadot/api";
 import { ApiTypes, AugmentedEvent, SubmittableExtrinsic } from "@polkadot/api/types";
 import { KeyringPair } from "@polkadot/keyring/types";
-import { Compact, u128, u16, u32, u64, Vec, Option, Bytes } from "@polkadot/types";
+import { Compact, u128, u16, u32, u64, Vec, Option } from "@polkadot/types";
```

Comment:
this was fixing some warnings that were muddying output when I was fixing compile errors.